routers/web/admin/users.go

```diff
@@ -332,6 +332,7 @@ func EditUserPost(ctx *context.Context) {
 
 	u.LoginName = form.LoginName
 	u.FullName = form.FullName
+	emailChanged := !strings.EqualFold(u.Email, form.Email)
```

Comment:
`form.Email` seems to be validated by go-chi's EmailPattern:
```
EmailPattern        = regexp.MustCompile(`\A[\w!#$%&'*+/=?^_`+"`"+`{|}~-]+(?:\.[\w!#$%&'*+/=?^_`+"`"+`{|}~-]+)*@(?:[\w](?:[\w-]*[\w])?\.)+[a-zA-Z0-9](?:[\w-]*[\w])?\z`)
```
Impossible to understand what it does.

I don't know whether it is correct, whether it produces the same result as `ValidateEmail`

Reply:
Yeah, we can discuss that but not block this PR?

Reply:
It's better to also use `ValidateEmail` here to match the behavior of API `EditUser`. 

LGTM.

Reply:
done.